.src/safe_utilities.cpp

```diff
@@ -45,10 +46,14 @@ namespace jluna
         #endif
 
         detail::_num_threads = n_threads;
-        if (julia_image_path.empty())
+        if (julia_bindir.empty() && image_path.empty())
```

Comment:
```suggestion
        if (julia_bindir.empty() and image_path.empty())
```
To be consistent with my style guide